packages/fx-core/src/core/question.ts

```diff
@@ -55,6 +58,33 @@ export const QuestionRootFolder: FolderQuestion = {
   title: "Workspace folder",
 };
 
+export const QuestionSelectTargetEnvironment: SingleSelectQuestion = {
+  type: "singleSelect",
+  name: CoreQuestionNames.TargetEnvName,
+  title: "Select the target environment",
+  staticOptions: [],
+  skipSingleOption: true,
+  forgetLastValue: true,
+};
+
+export const QuestionNewTargetEnvironmentName: TextInputQuestion = {
+  type: "text",
+  name: CoreQuestionNames.NewTargetEnvName,
+  title: "New target environment name",
```

Comment:
nit: I would prefer to remove "target" for the new environment scenario, just "New environment name".

Reply:
Yeah that looks better. Thanks for the suggestion.

And just would like to clarify that, the user experience will be reviewed with UX designer, and the current behavior is for demo purpose and not finalized. So it might be changed in future.